multiqc/modules/seqera_cli/seqera_cli.py

```diff
@@ -0,0 +1,218 @@
+""" MultiQC module to parse output from the Seqera Platform CLI """
+
+import datetime
+import json
+import logging
+import os
+import tarfile
+from collections import defaultdict
+
+from multiqc.modules.base_module import BaseMultiqcModule, ModuleNoSamplesFound
+from multiqc.plots import bargraph
+
+log = logging.getLogger(__name__)
+
+
+def _read_json_from_tar_gz(tar_file, fname):
+    try:
+        fh = tar_file.extractfile(tar_file.getmember(fname))
+        contents = fh.read()
+    except Exception as e:
+        log.warning(f"Could not extract file {fname} from archive {tar_file}: {e}")
+        return {}
+    try:
+        data = json.loads(contents)
+    except Exception as e:
+        log.warning(f"Could parse JSON from {fname} in {tar_file}: {e}")
+        return {}
+    return data
+
+
+def _parse_workflow_json(data):
+    keys = ["repository", "start", "complete", "revision"]
+    data = {k: data.get(k) for k in keys}
+    # "start" and "complete" are time stamps like time stamps like 2023-10-22T14:39:01Z
+    # parse them with a library, take the difference "complete" - "start" to get the
+    # duration, and convert the duration it to a human-readable format.
+    if "start" in data and "complete" in data:
+        start = datetime.datetime.strptime(data["start"], "%Y-%m-%dT%H:%M:%SZ")
+        complete = datetime.datetime.strptime(data["complete"], "%Y-%m-%dT%H:%M:%SZ")
+        data["duration"] = complete - start
+    return data
+
+
+def _parse_workflow_load_json(data):
+    keys = [
+        "cpuEfficiency",
+        "memoryEfficiency",
+        "cpuTime",
+        "readBytes",
+        "writeBytes",
+        "cost",
+        "pending",
+        "submitted",
+        "running",
+        "succeeded",
+        "failed",
+        "cached",
+    ]
+    data = {k: data.get(k) for k in keys}
+    return data
+
+
+class MultiqcModule(BaseMultiqcModule):
+    """
+    Seqera Platform CLI module for MultiQC. Should be able to process logs dump
+    usually written in a form of a tar-gz archive, as well as its uncompressed version.
+    that is reading workflow.json and workflow-load.json files directly.
+    """
+
+    def __init__(self):
+        super(MultiqcModule, self).__init__(
+            name="Seqera Platform CLI",
+            anchor="seqera_cli",
+            href="https://github.com/seqeralabs/tower-cli",
+            info="reports statistics generated by the Seqera Platform CLI.",
+            doi="10.1016/j.ajhg.2017.01.017",
```

Comment:
This is showing the citation for Peddy 😅 

I wonder if we can have some kind of CI check that looks for duplicated DOIs. Might be some valid reasons, but I bet that most would be accidental copy+paste artefacts.
```suggestion
            # doi="",  # No DOI for this tool
```